lib/src/builder/codegen/component_generator.dart

```diff
@@ -168,13 +165,10 @@ class _ComponentGenerator extends ComponentGenerator {
 
   @override
   String get defaultConsumedPropsImpl =>
-      r'List<ConsumedProps> get $defaultConsumedProps => ' +
-      declaration.props.switchCase(
-        (a) => 'const []', // Concrete props classes do not have generated meta
-        // Use propsMeta.forMixin instead of directly accessing the props class so that
-        // we don't reference the generated mixin and have to emit another warning comment.
-        (b) => '[propsMeta.forMixin(${propsNames.consumerName})]',
-      );
+      '  /// The default consumed props, comprising all props mixins used by ${propsNames.consumerName}.\n'
+      '  /// Used in -`ConsumedProps` methods if [consumedProps] is not overridden.\n'
```

Comment:
```suggestion
      '  /// Used in `ConsumedProps` methods if [consumedProps] is not overridden.\n'
```

Reply:
Addressed in e3f27c9c597555e0e3cb917a34687c9b9859fd57